problems/0-introduction/problem.md

```diff
@@ -18,7 +18,7 @@ When you succeed, you type `learnyouw3up` again and select the next exercise.
 
 Rinse and repeat for each excercise. Got it? Good.
 
-Ok, now I need you to make that JS file we talked about. For this exercise we're going to print out your DID. Yeah, like an ID, but DECENTRALIZED. Woof.
+Ok, now I need you to make that JS file we talked about, and call it `program.mjs`. For this exercise we're going to print out your DID. Yeah, like an ID, but DECENTRALIZED. Woof.
```

Comment:
Hmm, just below it says "When you're done, verify your solution with `learnyouw3up verify ex1.mjs`...".

Perhaps we should change "To verify your program, run: `$ADVENTURE_NAME verify program.mjs`" instead (so it reads `ex1.mjs`)? We'd need to change for every problem.

Reply:
great point. I didn't recognize the pattern until after I submitted this and I'd gotten to ex2 and beyond

Reply:
I updated the PR to change this to saying 'and call it `ex1.mjs`' and change the bottom from `program.mjs` to `ex1.mjs`. And I changed the bottom of all the other problem.md that said `program.mjs`